docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -86,7 +86,7 @@ jobs:
 
 ```yaml
 semgrep:
-  image: returntocorp/semgrep-agent:v1
+  image: returntocorp/semgrep-agent:v1 TODO
```

Comment:
Please, what should these configs look like now? Just `returntocorp/semgrep`?

The `:v1` is there to demonstrate how to use only a specific version of semgrep-agent? Is this recommended? @brendongo @underyx

Reply:
Nope, that's not correct. A full github actions config would look like:

```
name: Semgrep
on: [pull_request]
jobs:
  semgrep:
    name: Scan
    runs-on: ubuntu-20.04
    env:
      SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
    container:
      image: returntocorp/semgrep
    steps:
      - uses: actions/checkout@v3
      - run: semgrep ci
```

two things are important here:

1. the image changes to `returntocorp/semgrep`
2. the command changes to `semgrep ci`